Update input-method-editor-error.md (#6191)
MicrosoftDocs/windowsserverdocs: WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md

---
# Input method editor issue in RemoteApp scenarios

> The following issues have been addressed in Windows 10+ operating systems as of October 2021.

Reviewer: ```suggestion
The following issues have been addressed in Windows 10 and later as of October 2021.

```

Affected platforms:

- Azure Virtual Desktop
- Remote Desktop Services

> Please Note:

Reviewer: Delete this line. We don't say "please" in documentation.

Author: Approved

> 

Reviewer: >[!NOTE]

Author: Approved

> - Windows 8.1 and below are out of service and remain affected

Reviewer: ```suggestion
> - Windows 8.1 and earlier are out of service and remain affected.

```

> - Fix must be applied on both the Remote Desktop Server and Client machines

Reviewer: ```suggestion
- You must apply the update to both the Remote Desktop server and the client devices.

```
